assets/sass/components/surveys/_googlesitekit-surveys-modal.scss

```diff
@@ -24,7 +24,7 @@
 
 	// The max-width of the survey element is 400px, so we allow it to break
 	// away from being "centered" after 400px.
-	@media (min-width: 400px) {
+	@media (min-width: $width-xsmall + px) {
```

Comment:
D'oh, looks like we already have `$bp-xsmallOnly`. We can use that breakpoint and avoid this usage, which isn't used anywhere else in the plugin.